app/features/lfg/lfg-types.ts

```diff
@@ -45,3 +45,109 @@ type MinTierFilter = {
   _tag: "MinTier";
   tier: TierName;
 };
+
+const typeToNum: Map<LFGType, number> = new Map([
+  ["PLAYER_FOR_TEAM", 0],
+  ["TEAM_FOR_PLAYER", 1],
+  ["TEAM_FOR_COACH", 2],
+  ["COACH_FOR_TEAM", 3],
+]);
```

Comment:
What happens if a new `LFGType` is added? IIRC If we use `Record<LFGType, number>` here instead of a map it will check for us a build time we are handling all the cases.

Reply:
Yeah I guess adding an array called `LFGTYPES` like for `TIERS` would work. Both cases are similar, for the tiers I'm hoping whenever a new one is added the array is updated. Didn't find a way to iterate over all cases of a type sadly, that'd be a nice ts feature :(